scripts/contentScript.js

```diff
@@ -108,11 +138,20 @@ async function getHighlightedPR(repoOwner, reponame, userId) {
 	}
 };
 
+// for showing all tracked/ untrack pr in a organization
+async function getTrackedPR(orgName) {
+	const body = { "org": `${orgName}` }
+	const url = 'https://gcscruncsql-k7jns52mtq-el.a.run.app/setup/repos';
```

Comment:
I think we should make a global constant defining the base URL of the backend at one place.